src/zappy_gui_src/Core/Animation/Animation.hpp

```diff
@@ -0,0 +1,100 @@
+/*
+** EPITECH PROJECT, 2022
+** zappy
+** File description:
+** Animation
+*/
+
+/// \file src/zappy_gui_src/Core/Animation/Animation.hpp
+
+#ifndef ANIMATION_HPP_
+#define ANIMATION_HPP_
+
+#include <SFML/Graphics.hpp>
+
+namespace gui {
+    /// \brief Animation class should be used to create an animation in sfml.
+    /// \note It contains a list of texture that will be updated, depending on a duration time, with the function update.
+    /// You should define a duration between each frame.
+    /// You should define a position for the animation.
+    /// You should define a size for the animation.
+    /// You should add a texture for the animation.
+    /// \warning Be careful, a texture is always add at the end of the vector of texture.
```

Comment:
warning and be careful means the same things

Reply:
\warning is a balise for the documentation generation, and be careful is the begging of the sentance

Reply:
that's why be carefull isn't useful here